src/aind_data_access_api/document_db.py

```diff
@@ -235,7 +238,7 @@ def retrieve_docdb_records(
         sort: Optional[dict] = None,
         limit: int = 0,
         paginate: bool = True,
-        paginate_batch_size: int = 10,
+        paginate_batch_size: int = 1000,
```

Comment:
Updated default batch size to 1000 after testing. Each request for ~1000 records takes ~1s.

Reply:
Depending on whose PR is merged first, we also need to change the default batch size in Micah's `retrieve_schema_records` PR.